src/main/java/com/hmrc/hrsystem/model/User.java

```diff
@@ -50,13 +58,31 @@ public void setAge(int age) {
         this.age = age;
     }
 
+    public BigDecimal getSalary() {
+        return salary;
+    }
+
+    public void setSalary(BigDecimal salary) {
+        this.salary = salary;
+    }
+
+    public Gender getGender() {
+        return gender;
+    }
+
+    public void setGender(Gender gender) {
+        this.gender = gender;
+    }
+
```

Comment:
kindly add equals & hashcode methods as well.

Reply:
Can we cover this point in next changes.